pkg/cli/cliflags/flags.go

```diff
@@ -780,4 +780,34 @@ in the history of the cluster.`,
 		Name:        "align",
 		Description: `Align the output.`,
 	}
+
+	LogDir = FlagInfo{
+		Name: "log-dir",
+		Description: `
+If non-empty, write log files in this directory. If empty, write log files to
+the <store-dir>/logs where <store-dir> is the directory of the first on disk
```

Comment:
I'd remove `the` from `the <store-dir>/logs`.

Reply:
Done.